api/python/provisioner/commands/pillar_translate.py

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com."
+#
+
+import logging
+import json
+from pathlib import Path
+from collections.abc import Mapping
+
+from ._basic import RunArgs, RunArgsBase
+
+from ..config import (
+    ALL_MINIONS,
+    CORTX_CONFIG_DIR,
+    CONFSTORE_CLUSTER_CONFIG
+)
+
+from ..vendor import attr
+
+from . import (
+    PillarGet
+)
+
+logger = logging.getLogger(__name__)
+
+
+@attr.s(auto_attribs=True)
+class RunArgsPillarTranslate(RunArgsBase):
+    local: bool = RunArgs.local
+
+
+# Method to address ConfStore LIMITATION on accepting ONLY string values
+# Can be commented when the bug is addressed.
+
+# Converts all values(null, bool, int) to string
+
+def convert_to_str(obj, repl=''):
```

Comment:
Should be a class method and not a function. We are not calling it as default method from json.dump, so not required to be a function.

Reply:
Comment addressed. Thanks